### PR TITLE
Update design-proposals URL

### PR DESCRIPTION
**Pull request by ping035627**
What type of PR is this?
/kind cleanup

What this PR does / why we need it:
The URL of design-proposals has changed, update it.

Which issue(s) this PR fixes:

Fixes #

Special notes for your reviewer:

Does this PR introduce a user-facing change?:

Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

/release-note-none
/priority backlog
/triage accepted

### CODE REVIEW

**Review by 249043822 — APPROVED** [review]
/lgtm

**Review by mrunalp — APPROVED** [review]

File: pkg/kubelet/apis/config/v1beta1/defaults.go [hunk 1]
```diff
@@ -44,7 +44,7 @@ const (
 var (
 	zeroDuration = metav1.Duration{}
 	// TODO: Move these constants to k8s.io/kubelet/config/v1beta1 instead?
-	// Refer to [Node Allocatable](https://git.k8s.io/community/contributors/design-proposals/node/node-allocatable.md) doc for more information.
+	// Refer to [Node Allocatable](https://github.com/kubernetes/design-proposals-archive/blob/main/node/node-allocatable.md) doc for more information.
```
Comment (thockin) [inline pkg/kubelet/apis/config/v1beta1/defaults.go:47]: https://git.k8s.io/design-proposals-archive/node/node-allocatable.md

File: pkg/kubelet/kuberuntime/kuberuntime_container_windows.go [hunk 2]
```diff
@@ -44,7 +44,7 @@ func (m *kubeGenericRuntimeManager) applyPlatformSpecificContainerConfig(config
 }
 
 // generateWindowsContainerConfig generates windows container config for kubelet runtime v1.
-// Refer https://github.com/kubernetes/community/blob/master/contributors/design-proposals/node/cri-windows.md.
+// Refer https://github.com/kubernetes/design-proposals-archive/blob/main/node/cri-windows.md.
```
Comment (thockin) [inline pkg/kubelet/kuberuntime/kuberuntime_container_windows.go:47]: https://git.k8s.io/design-proposals-archive/node/cri-windows.md

**Comment by ping035627** [comment]
@thockin I have updated it, review again, thanks!

**Review by thockin — COMMENTED** [review]
Thanks!

/lgtm
/approve

**Pull request merged**